Marlin/src/lcd/extensible_ui/lib/lulzbot/ftdi_eve_lib/extended/sound_player.cpp

```diff
@@ -38,7 +38,8 @@ namespace FTDI {
 
     #if ENABLED(TOUCH_UI_DEBUG)
       SERIAL_ECHO_START();
-      SERIAL_ECHOLNPAIR("Playing note ", int(note), ", instrument ", int(effect));
+      SERIAL_ECHOPAIR("Playing note ", int(note));
+      SERIAL_ECHOPAIR(", instrument ", int(effect));
```

Comment:
Are you using compatibility layer versions of these macros?
Looks like the second one should be `SERIAL_ECHOLNPAIR`.